src/pre-local.ts

```diff
@@ -1,6 +1,7 @@
 import { EsbuildPlugin } from '.';
 export function preLocal(this: EsbuildPlugin) {
   this.serviceDirPath = this.buildDirPath;
+  this.serverless.config.servicePath = this.buildDirPath;
   // Set service path as CWD to allow accessing bundled files correctly
   process.chdir(this.serviceDirPath);
```

Comment:
is this still required? invoke local from AWS serverless plugin seems to only use `this.serverless.config.servicePath`

Reply:
I am not sure - if you look at the code from the Webpack plugin there is still the `chdir`